client/src/components/Header.css

```diff
@@ -91,10 +91,28 @@ button:hover {
   display: flex;
   flex-direction: column;
   align-items: center;
+  margin-top: 30px;
 }
 
 h2 {
   font-size: 24px;
   margin-bottom: 20px;
 }
 
+.close-button {
+  position: absolute;
+  top: -190px;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
```

Comment:
The close button is a `button` element so you don't need to set the cursor. It doesn't make any difference but just so you know!